mod/groups/start.php

```diff
@@ -329,7 +329,7 @@ function groups_set_icon_url($hook, $type, $url, $params) {
 		return "groupicon/$group->guid/$size/$icontime.jpg";
 	}
 
-	return "mod/groups/graphics/default{$size}.gif";
```

Comment:
One problem is that if someone tries to override the image in a view, but the direct references are left, that could actually feel broken because it will show the "overridden" image.